MC/config/QC/json/qc-emc-async.json

```diff
@@ -0,0 +1,48 @@
+{
+  "qc": {
+    "config": {
+      "database": {
+        "implementation": "CCDB",
+        "host": "ccdb-test.cern.ch:8080",
+        "username": "not_applicable",
+        "password": "not_applicable",
+        "name": "not_applicable"
+      },
+      "Activity": {
+        "number": "42",
+        "type": "2",
+        "provenance": "qc_mc",
+        "passName": "passMC",
+        "periodName": "SimChallenge"
+      },
+      "monitoring": {
+        "url": "infologger:///debug?qc"
+      },
+      "consul": {
+        "url": "http://consul-test.cern.ch:8500"
+      },
+      "conditionDB": {
+        "url": "ccdb-test.cern.ch:8080"
+      }
+    },
+    "tasks": {
+      "DigitsTask": {
+        "active": "true",
+        "className": "o2::quality_control_modules::emcal::DigitsQcTask",
+        "moduleName": "QcEMCAL",
+        "detectorName": "EMC",
+        "cycleDurationSeconds": "60",
+        "maxNumberCycles": "-1",
+        "dataSource": {
+          "type": "direct",
+          "query": "emcal-digits:EMC/CELLS;emcal-triggerecords:EMC/CELLSTRGR"
+        },
+        "taskParameters": {
+          "nothing": "rien"
+        }
```

Comment:
```suggestion
        }
```

Reply:
no neede for this

Reply:
OK, currently the digits task has no task parameters, I will remove the part